lib/src/presenter/pages/portfolio/widgets/monetary_infos_crypto_in_tile.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:intl/intl.dart';
+
+import '../../../../domain/entities/wallet_crypto_entity.dart';
+import '../../../controllers/visibility/visibility_provider.dart';
+
+class MonetaryInfosCryptoInTile extends HookConsumerWidget {
+  const MonetaryInfosCryptoInTile({
+    Key? key,
+    required this.walletCryptoEntity,
+  }) : super(key: key);
+
+  final WalletCryptoEntity walletCryptoEntity;
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final visibility = ref.watch(visibilityProvider);
+
+    return Row(
+      children: [
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.end,
+          children: [
+            Visibility(
```

Comment:
É possível componentizar esse Visibility, passando os valores variáveis como parâmetros ou, se for componentizado, também é possível obter o tamanho do Text para utilizar no container caso queira algo mais específico.

[link para pegar size do Text](https://stackoverflow.com/questions/52659759/how-can-i-get-the-size-of-the-text-widget-in-flutter)